kornia/augmentation/container/patch.py

```diff
@@ -122,7 +123,8 @@ def __init__(
         super().__init__(
             *args, same_on_batch=same_on_batch, return_transform=False, keepdim=keepdim, random_apply=_random_apply
         )
-        assert padding in ["same", "valid"], f"`padding` must be either `same` or `valid`. Got {padding}."
+        if padding not in ["same", "valid"]:
```

Comment:
Tuples take up less RAM than lists because they are read-only and they are a signal to the reader that the contents will not change during runtime.

$ `python3`
```
>>> import sys
>>> sys.getsizeof(["same", "valid"])
72
>>> sys.getsizeof(("same", "valid"))
56
```